src/plugins/management/public/components/management_app/management_app.tsx

```diff
@@ -114,30 +108,28 @@ export const ManagementApp = ({
   };
 
   return (
-    <RedirectAppLinks coreStart={dependencies.coreStart}>
-      <I18nProvider>
+    <KibanaRenderContextProvider i18n={coreStart.i18n} theme={coreStart.theme}>
```

Comment:
I placed `KibanaRenderContextProvider` above other context providers, in all changes, for consistency